spec/draft/extensions/sparse_interchange.rst

```diff
@@ -0,0 +1,39 @@
+.. _sparse_interchange:
+
+Sparse interchange
+==================
+
+    Array API specification for sparse interchange functions.
+
+Extension name and usage
+------------------------
+
+If implemented, this extension must be retrievable via::
+
+    >>> if hasattr(x, '__dlpack__'):
```

Comment:
Should this be `__binsparse__`?

Reply:
In general, what is documented here diverges from how we've exposed `fft` and `linalg` extensions. Namely, these come from the namespace, not array object.